icons/triangle-right.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
:-1: don't like it.

Reply:
> 👎 don't like it.

@jguddas Maybe useful to have _slightly_ more constructive feedback than that 😅

Do we not want to cover all base `shapes`…? Should be part of #1142?